src/PayU/Alu/Billing.php

```diff
@@ -119,6 +124,16 @@ public function withIdentityCardSeries($identityCardSeries)
     }
 
     /**
+     * @param string $identityCardType
+     * @return $this
```

Comment:
Indeed. The existing pattern is bad everywhere. How it is right now works in PHPStorm for autocomplete but other developers might not be that lucky.
I guess it would be better to have a separate task for changing it.
Let's leave it like this for this current pull request.